src/ServiceStack/Testing/MockHttpResponse.cs

```diff
@@ -16,6 +16,8 @@ public MockHttpResponse(IRequest request = null)
             this.Headers = PclExportClient.Instance.NewNameValueCollection();
             this.OutputStream = new MemoryStream();
             this.TextWritten = new StringBuilder();
+            if (HostContext.AppHost == null)
+                throw new NullReferenceException("A ServiceStackHost has not been initialised in the test.");
             this.Cookies = HostContext.AppHost.GetCookies(this);
```

Comment:
Can you instead just change it to: 

```csharp
this.Cookies = HostContext.AssertAppHost().GetCookies(this);
```

That way we don't need to embed the same error messages everywhere.